packages/lesmis/src/views/AdminPanel/SurveyResponsesView.js

```diff
@@ -78,7 +78,7 @@ export const DraftSurveyResponsesView = props => (
   <SurveyResponsesPage
```

Comment:
Comment about lines 50 - 74: Could you please change the reject button from a straight delete to a mutation of `{ approval_status: 'rejected' }`?

Reply:
Good idea. I also updated the all survey responses tab to now be rejected so there is a nice three tab flow (Under Review, Approved, Rejected)